docs/api-reference/geo-layers/tile-layer.md

```diff
@@ -114,11 +114,24 @@ If supplied, `getTileData` is called to retrieve the data of each tile. It recei
 - `z` (Number) - z index of the tile
 - `url` (String) - resolved url of the tile if the `data` prop is provided, otherwise `null`
 - `bbox` (Object) - bounding box of the tile. When used with a geospatial view, `bbox` is in the shape of `{west: <longitude>, north: <latitude>, east: <longitude>, south: <latitude>}`. When used with a non-geospatial view, `bbox` is in the shape of `{left, top, right, bottom}`.
+- `signal` (Object) - an [AbortSignal](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal) that may be signalled if there are too many queued requests. Note: only tiles that aren't visible will be aborted.
 
 It should return either the tile data or a Promise that resolves to the tile data.
 
 This prop is not required if `data` points to a supported format (JSON or image by default). Additional formats may be added by registering [loaders.gl](https://loaders.gl/modules/core/docs/api-reference/register-loaders) modules.
 
+It is recommended to pass `signal` to any `fetch` calls and check its `aborted` property before doing any expensive computation.
+
+```js
+const {signal} = tile;
+const data = fetch(url, {signal});
+
+if (signal.aborted) {
+   return null;
```

Comment:
Maybe add a sentence here saying "If you check for `signal.aborted` and interrupt your processing, you should return `null` so that incorrect data is not cached" ?

Reply:
They can return falsy or throw. Either way it will not be cached.

Reply:
But I can spell that out in the docs if you think it needs it.

Reply:
Yes you're right. Maybe "If you abort `getTileData`, don't return incomplete data or it will be cached". I think one such sentence in the docs would be helpful.